app/middleware/get_gateway_account.js

```diff
@@ -35,9 +35,9 @@ module.exports = function (req, res, next) {
         return connectorClient.getStripeAccountSetup(accountId, req.correlationId)
       }
     })
-    .then((accountSetupDetails = null) => {
-      if (accountSetupDetails) {
-        req.account.accountSetupDetails = accountSetupDetails
+    .then((connectorGatewayAccountStripeProgress = null) => {
+      if (connectorGatewayAccountStripeProgress) {
+        req.account.connectorGatewayAccountStripeProgress = connectorGatewayAccountStripeProgress
```

Comment:
As this changes the name of the attribute passed through to the template I think it will need to be updated everywhere https://github.com/alphagov/pay-selfservice/blob/7187cf8978ce68c72ed989f20e6367b87a8dbc2c/app/views/includes/phase_banner.njk#L9

Reply:
Good Spot - done